src/network/network_admin.cpp

```diff
@@ -336,8 +335,15 @@ NetworkRecvStatus ServerNetworkAdminSocketHandler::SendCompanyInfo(const Company
 		p->Send_uint8(owner);
 	}
 
+
+	uint8 rating = std::count_if(Company::Iterate().begin(), Company::Iterate().end(), [&](auto company){
```

Comment:
a bit lower is a function `SendCompanyUpdate`, which sends the same info as this packet, minus some immutable fields (no clue how we ever were like: yeah, that is a great idea! Let's have two nearly identical packets, but .. here we are :D). As rating can change, and if we go for this change, I think this code should also be added there. Just to avoid confusion with any admin-port user :)